libnavigation-base/src/main/java/com/mapbox/navigation/base/formatter/DistanceFormatterOptions.kt

```diff
@@ -73,17 +74,18 @@ class DistanceFormatterOptions private constructor(
      */
     class Builder(applicationContext: Context) {
         private val applicationContext: Context = applicationContext.applicationContext
-        private var unitType: String = VoiceUnit.UNDEFINED
-        private var locale: Locale? = null
+        private var locale: Locale = applicationContext.inferDeviceLocale()
+        private var unitType: UnitType? = null
         private var roundingIncrement = Rounding.INCREMENT_FIFTY
 
         /**
-         * Policy for the various units of measurement, UNDEFINED uses default for locale country
+         * Policy for the various units of measurement.
+         * [UnitType] based on [Locale] is used by default.
          *
-         * @param unitType String
+         * @param unitType UnitType
          * @return Builder
          */
-        fun unitType(@VoiceUnit.Type unitType: String) =
+        fun unitType(unitType: UnitType) =
```

Comment:
```suggestion
        fun unitType(unitType: UnitType?) =
```
For completeness. If there's a default type, we should be able to make this value `null`.

Reply:
done